app/workers/perform_user_action_worker.rb

```diff
@@ -1,8 +1,8 @@
-class PerformActionWorker
+class PerformUserActionWorker
```

Comment:
Renaming this will mean that old jobs currently in the queue will break. We'll have to do this in two phases where we make a copy of this, and start only pushing new jobs for the new name. Then a follow-up PR can remove the old unused worker once all jobs for it are out of the queue.